sphinx_copybutton/__init__.py

```diff
@@ -30,6 +30,12 @@ def add_to_context(app, config):
     config.html_context.update(
         {"copybutton_copy_empty_lines": config.copybutton_copy_empty_lines}
     )
+    config.html_context.update(
+        {"copybutton_end_of_line_character": config.copybutton_end_of_line_character}
```

Comment:
For the sake of "naming things", we might want to adjust this to `copybutton_line_continuation_character`?

```suggestion
        {"copybutton_line_continuation_character": config.copybutton_line_continuation_character}
```